### PR TITLE
Django - Email Sending

### PR DESCRIPTION
**Pull request by NorakGithub**

### CODE REVIEW

**Comment by powerdefy** [comment]
When testing in local , I think using MailHog or other email server is the better way to prevent sending email.  What do you think krub?

File: django/email/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+from django.core.mail import send_mail
+send_mail(
+    'Subject',
+    'Message',
+    allowed_receivers,
+    fail_silently=False
+)
+```
+
```
Comment (powerdefy) [inline django/email/README.md:55]: Should we mention that Sendgrid not allow duplicate email in `receivers` krub?

**Comment by earthpyy** [comment]
> When testing in local , I think using MailHog or other email server is the better way to prevent sending email. What do you think krub?

I do agree with P' @powerdefy krubbb.

**Comment by NorakGithub** [comment]
@powerdefy Will add krub

**Review by powerdefy — APPROVED** [review]

**Review by earthpyy — APPROVED** [review]
Thanks krab

**Pull request merged**